src/Compute/Compute.Test/ScenarioTests/DiskRPTests.ps1

```diff
@@ -1825,4 +1825,46 @@ function Test-SnapshotConfigTierOptionEnhancedSpeed
         # Cleanup
         Clean-ResourceGroup $rgname
     }
+}
+
+<#
+.SYNOPSIS
+Using a TL disk, use the SecureVmGuestStateSas parameter to get the securityDataAccessSAS value.
+#>
+function Test-DiskGrantAccessGetSASWithTL
+{
+    $rgname = Get-ComputeTestResourceName;
+	$loc = Get-ComputeVMLocation;#'eastus';
+
+	try
+    {
+        # $rgname = "adsandtlg4";
+        # $loc = "eastus";
+		New-AzResourceGroup -Name $rgname -Location $loc -Force;
+
+        $diskname = "d" + $rgname;
+        $securityTypeTL = "TrustedLaunch";
```

Comment:
not needed. Could consider a test for a nonTL disk, but not specific to PS functionality more a test that the backend is returning what we expect.